docs/Get-HelpPreview.md

```diff
@@ -5,25 +5,25 @@ schema: 2.0.0
 
 # Get-HelpPreview
 ## SYNOPSIS
-Preview the output Get-Help would return from an external help file(s).
+Displays your generated external help as **Get-Help** output.
+
 ## SYNTAX
 
 ```
-Get-HelpPreview -Path <String[]> [<CommonParameters>]
```

Comment:
I don't remember removing them either - If it's my mistake, great! Then we know what's going on.